files/image_config/rsyslog/rsyslog.d/00-sonic.conf

```diff
@@ -1,15 +1,12 @@
 ## Quagga rules
 
-if $programname == ["bgp#quagga",
-                    "bgp#watchquagga",
-                    "bgp#zebra"]
-    then {
-    /var/log/quagga/zebra.log
+if re_match($programname, "bgp[0-9]*#(frr|zebra)") then {
```

Comment:
- What is the plan for staticd log target?
- With single ASIC system, do we have a number after bgp as programname?

Reply:
Hi @zhenggen-xu I have updated to handle the staticd and watchfrr logs, direct it to zebra.log.
In case of single ASIC system, there will not be a number it will be bgp#bgpd

Reply:
thanks, in that case, the re_match condition should be changed to support single ASIC system too?

Reply:
This match condition matches the single asic scenario as well. I verified and added couple of logs in the description, please take a look.